cmd/agent/app/reporter/flags.go

```diff
@@ -35,15 +37,18 @@ type Type string
 // Options holds generic reporter configuration.
 type Options struct {
 	ReporterType Type
+	AgentTags    Type
```

Comment:
I think this should be map[string]string

Reply:
+1 to `[]model.Tag` as per your comment below.